dkw22/code-challenge
# pull request between feature and master branch

## Overview

This pull request is my submission of the coding challenge for the DataMade Developer Role. It is a simple recreation of the Address Parser.

## Testing Instructions

* Testing this PR is quite simple.
* Be sure to clone this branch before trying to test.
* Make sure to have Docker and Docker Compose Installed Locally.
* Once those are installed, you'll have to build the appicaltion containers:
`docker-compose build`
* Then run the app:
`docker-compose up`
* The app will log to the console, and you should be able to visit it at http://localhost:8000/.
* The website application should then be fully functional to use.
* Some test strings that you can use are:
* ✅ Valid: 123 main st chicago il
* ❌ Invalid: 123 main st chicago il 123 main st
* Lastly, have fun!
